tvb_documentation/sim_doc/img/classes_monitors.svg

```diff
@@ -17,7 +17,7 @@
 <text text-anchor="start" x="1344.5" y="-371.3" font-family="Times,serif" font-size="14.00">hemodynamic_response_function</text>
 <text text-anchor="start" x="1344.5" y="-356.3" font-family="Times,serif" font-size="14.00">hrf_length : Float</text>
 <text text-anchor="start" x="1344.5" y="-341.3" font-family="Times,serif" font-size="14.00">period : Float</text>
-<text text-anchor="start" x="1344.5" y="-326.3" font-family="Times,serif" font-size="14.00">hrf_kernel : HRFKernelEquation</text>
+<text text-anchor="start" x="1344.5" y="-326.3" font-family="Times,serif" font-size="14.00">equation : HRFKernelEquation</text>
```

Comment:
This is not a file to change manually. Did you run `class_diagrams.sh` ?

Reply:
No, I just renamed it everywhere.